### PR TITLE
Helperlist

### PR DESCRIPTION
**Pull request by JohannesMisch**

### CODE REVIEW

File: components/AddDeviceForm.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  height: 90%;
+  width: 85%;
+  top: 40px;
+  left: 50%;
+  transform: translate(-50%, 0);
+  display: flex;
+  flex-direction: column;
+  padding: 10px 50px;
+  gap: 10px;
+`;
```
Comment (Long-HaHoang) [inline components/AddDeviceForm.js:98]: Nice refactor.
Seems like you move it to it's own component 👍

File: components/Card.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default function Card({
     <>
       <section>
         {CATEGORY_MAP[deviceCategory]}
-        <p>Device:{name.toUpperCase()}</p>
+        <p>Device:{name}</p>
```
Comment (Long-HaHoang) [inline components/Card.js:34]: Well if you don't like the toUpperCase you can also make the first letter to a capital letter.
```javascript
`${name.splice(0,1).toUpperCase()}${name.splice(1).toLowerCase()}`
```

File: db.json [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
-    "id": 0,
     "device": "TV",
     "device_category": "Entertainment",
     "model": "LG 55NANO796PC LED TV",
     "power_consumption": 95,
     "power_consumption_standby": 0.5,
     "average_usage_time": 4,
-    "location": "Livingroom"
+    "location": "Livingroom",
+    "id": 0
```
Comment (Long-HaHoang) [inline db.json:10]: what's point of moving the id?

File: pages/_app.js [hunk 4]
```diff
@@ -12,14 +12,15 @@ export default function App({ Component, pageProps }) {
     setDevices((oldDevices) => [
       {
         ...newDevice,
+        device: newDevice.device.toUpperCase(),
```
Comment (Long-HaHoang) [inline pages/_app.js:15]: Ah I see. You adding the .toUpperCase while setting. 👍
But what about your `Clothes dryer` and `MacBook Air` from your JSON.

File: pages/add-device.js [hunk 5]
```diff
@@ -94,14 +58,23 @@ export default function AddNewDevice({ createDevice }) {
...
-  gap: 10px;
-`;
+
 const StyledButton = styled.button`
+  z-index: 3;
   position: fixed;
   bottom: 20px;
   left: 20px;
+  background-color: red;
+`;
```
Comment (Long-HaHoang) [inline pages/add-device.js]: don't you want to make it bigger.
I know it's now red to get the attentions.
But you can still overlook it.

File: pages/index.js [hunk 6]
```diff
@@ -30,6 +32,11 @@ export default function Home({ devices, handleDelete }) {
           <option value="Appliances">Appliances</option>
           <option value="Work">Work</option>
           <option value="Lighting">Lighting</option>
+          <option value="">--Please choose an option--</option>
```
Comment (Long-HaHoang) [inline pages/index.js]: Why do you return `--Please choose an option--`?
Now you have this option twice.

**Review by Long-HaHoang — CHANGES_REQUESTED** [review]
Some thinks I noticed.
More in the comments.

File: db.json (hunk 3 above)
Comment (JohannesMisch) [reply db.json:10]: just some late-night stuff... no real point.

**Review by Long-HaHoang — APPROVED** [review]
Well besides the double option. It's fine by me.

File: pages/index.js (hunk 6 above)
Comment (JohannesMisch) [reply pages/index.js]: the options below the second "Please choose..." do not work. just forgot to delete them.

File: pages/add-device.js (hunk 5 above)
Comment (JohannesMisch) [reply pages/add-device.js]: Yeah sure if you want to i totally can make the button bigger

File: components/Card.js (hunk 2 above)
Comment (JohannesMisch) [reply components/Card.js:34]: Nice, thanks for the suggestion. Your initial Idea to make the name `toUpperCase()` is great I want to keep it that way.
I just needed it to be all capital letters when I save them to the local storage to correctly compare them later on.

**Pull request merged**